drivers/windows/R/bootstrap.R

```diff
@@ -1,31 +1,68 @@
-bootstrap_update <- function(development = NULL, root = NULL) {
+bootstrap_update <- function(development = NULL, root = NULL,
+                             platform = "windows") {
```

Comment:
move platform first and drop default arg

Reply:
Done